src/pages/settings/Subscription/SubscriptionSettings/index.tsx

```diff
@@ -18,14 +19,16 @@ function SubscriptionSettings() {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const theme = useTheme();
+    const subscriptionPlan = useSubscriptionPlan();
     const [privateSubscription] = useOnyx(ONYXKEYS.NVP_PRIVATE_SUBSCRIPTION);
 
+    const isCollect = subscriptionPlan === CONST.POLICY.TYPE.TEAM;
+
     // TODO these default state values will come from API in next phase
     const [autoRenew, setAutoRenew] = useState(true);
     const [autoIncrease, setAutoIncrease] = useState(false);
 
-    // TODO this will come from API in next phase
-    const expirationDate = format(new Date(), CONST.DATE.MONTH_DAY_YEAR_ABBR_FORMAT);
+    const autoRenewalDate = privateSubscription?.endDate ? format(addMonths(new Date(privateSubscription?.endDate), 1), CONST.DATE.MONTH_DAY_YEAR_ABBR_FORMAT) : '';
```

Comment:
Sorry, I gave you bad info regarding adding a month to the end date. The autoRenewalDate should just be equal to the endDate (keep the formatting you've applied here though)

Reply:
I think we need to tweak what is passed to the Date constructor? Ultimately it should say it renews on June 1, 2025 in this case
![Screenshot 2024-06-07 at 9 56 42 AM](https://github.com/Expensify/App/assets/7277067/0261a0db-56bf-459b-9d54-1006bd48094d)

Reply:
hmm, checking

Reply:
Thats weird, because it is working fine on my end

<img width="1748" alt="image" src="https://github.com/Expensify/App/assets/23487612/145aba84-9fc1-45cb-924e-cf90ddcedb02">

Reply:
I think `const autoRenewalDate = privateSubscription?.endDate ? format(new Date(`${privateSubscription?.endDate}T00:00:00`), CONST.DATE.MONTH_DAY_YEAR_ABBR_FORMAT) : '';` would work, currently it's picking up my local timezone I think

Reply:
Ok, I've changed it. I will be finishing for today but on monday I might get back to this (in this or one of next PRs) and refactor it to something more elegant. I believe `getNewSubscriptionRenewalDate` might have the same TZ issue so I will look into it

Reply:
Okay, I'm going to merge this PR to keep things moving but definitely feel free to refactor in a follow up PR. Thank you!